slam_toolbox/src/loop_closure_assistant.cpp

```diff
@@ -125,48 +125,93 @@ void LoopClosureAssistant::publishGraph()
 /*****************************************************************************/
 {
   interactive_server_->clear();
-  std::unordered_map<int, Eigen::Vector3d>* graph = solver_->getGraph();
+  karto::MapperGraph * graph = mapper_->GetGraph();
 
-  if (graph->size() == 0)
+  if (!graph || graph->GetVertices().empty())
   {
     return;
   }
 
-  ROS_DEBUG("Graph size: %i",(int)graph->size());
```

Comment:
Please readd the debug statement

Reply:
Done in https://github.com/SteveMacenski/slam_toolbox/pull/467/commits/9f4eb994925ce5246694f4d76656802bba562ef5.